unpick-format-utils/src/main/java/daomephsta/unpick/constantmappers/datadriven/parser/v2/UnpickV2Remapper.java

```diff
@@ -31,30 +34,66 @@ public UnpickV2Remapper(Map<String, String> classMappings, Map<MethodKey, String
 	{
 		this.classMappings = classMappings;
 		this.methodMappings = methodMappings;
+		this.fieldMappings = Collections.emptyMap();
 		this.delegate = delegate;
```

Comment:
[NIT] Should really call the other constructor here.

Reply:
Yeah I also realised that after writing this, I can change it if you want to.

Reply:
Sure, easy enough to do